pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/tools/FieldSettingsTestUtil.java

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.dsm.util.tools;
+
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dao.settings.FieldSettingsDao;
+import org.broadinstitute.dsm.db.dto.settings.FieldSettingsDto;
+
+public class FieldSettingsTestUtil {
+
+    /**
+     * Creates a radio button field setting
+     * @return fieldSettingsId
+     */
+    public static int createRadioFieldSettings(String fieldType, String columnName, String possibeValues,
+                                                int ddpInstanceId) {
+        FieldSettingsDao fieldSettingsDao = FieldSettingsDao.of();
+        FieldSettingsDto.Builder builder = new FieldSettingsDto.Builder(ddpInstanceId);
+        FieldSettingsDto fieldSettings = builder.withFieldType("AT_PARTICIPANT_EXIT")
+                .withColumnName("EXITSTATUS")
```

Comment:
Might be worth a quick grep to see if these strings are already defined as constants somewhere.  `ATDefaultValues.AT_PARTICIPANT_EXIT` for example.